core/src/main/java/com/linecorp/armeria/common/StringMultimap.java

```diff
@@ -465,8 +470,8 @@ public final boolean containsObject(IN_NAME name, Object value) {
 
     @Override
     public final boolean containsBoolean(IN_NAME name, boolean value) {
-        final Boolean v = getBoolean(name);
-        return v != null && v == value;
+         return contains(name, val -> AsciiString.contentEqualsIgnoreCase(val, String.valueOf(value)) ||
+                                      AsciiString.contentEquals(val, value ? "1" : "0"));
```

Comment:
I'm unsure `tRUE` or `TrUe` is treated as a boolean value. 
How about only supporting the values defined in `stringToBoolean`?
```suggestion
         return contains(name, actual -> {
            final Boolean maybeBoolean StringUtil.toBoolean(actual, false);
            return maybeBoolean != null && maybeBoolean == value;
         });
```

Reply:
✅ updated, only support `TRUE, true, 1 / FALSE, false, 0` as boolean value~!